src/org.xtuml.bp.mc.masl/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MASL Export Builder
+Bundle-SymbolicName: org.xtuml.bp.mc.masl; singleton:=true
+Bundle-Version: 6.17.0.qualifier
+Bundle-Activator: org.xtuml.bp.mc.masl.Activator
+Bundle-Vendor: xtUML.org
+Require-Bundle: org.eclipse.ui,
+ org.eclipse.core.runtime,
+ org.eclipse.core.resources,
+ org.eclipse.core.variables,
+ org.xtuml.bp.core,
+ org.xtuml.bp.io.core,
+ org.xtuml.bp.mc,
+ org.eclipse.debug.ui,
+ org.eclipse.ui.ide,
+ org.xtuml.bp.ui.explorer,
+ org.xtuml.bp.utilities,
+ org.xtuml.bp.x2m
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ActivationPolicy: lazy
+Export-Package: org.xtuml.bp.mc.masl
+Bundle-ClassPath: masl.jar
+Eclipse-BundleShape: dir
```

Comment:
Add:
Automatic-Module-Name: org.xtuml.bp.mc.masl

One of the changes I have in my 8058 PR is to add this value to all the manifest files.  It resolves a warning.

Reply:
done